ci/requirements-dev-3.5.yml

```diff
@@ -29,6 +31,7 @@ dependencies:
   - ruamel.yaml
   - six
   - sqlalchemy>=1.0.0,<1.1.15
+  - thrift
```

Comment:
ame question as above

Reply:
I can try to remove that .. but I think pymad was breaking without thrift